packages/Pagination/src/components/ArrowItem/ArrowItem.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { getArrowIcon } from "./ArrowItem.styles";
+import * as sc from "./ArrowItem.styles";
+
+const propTypes = {
+  /* Description of the ArrowItem for assistive technology */
+  a11yText: PropTypes.string,
+
+  /** To set the ArrowItem as disabled */
+  isDisabled: PropTypes.bool,
+
+  /** Callback to be executed when the ArrowItem is clicked or activated by keyboard. */
+  onClick: PropTypes.func,
+
+  /** What type of arrow to display */
+  type: PropTypes.string,
+};
+
+const defaultProps = {
+  a11yText: null,
+  isDisabled: false,
+  onClick: () => {},
+  type: "right",
+};
+
+function ArrowItem(props) {
+  const { a11yText, isDisabled, onClick, type } = props;
+  const Icon = getArrowIcon(`Arrow-${type}`);
+  const I18n = useI18n();
+
+  return (
+    <sc.ArrowItem aria-label={a11yText || I18n.t(`pagination.${type}`)} isDisabled={isDisabled} onClick={onClick}>
```

Comment:
This might fail accessibility test, it should be a RawButton or a Button, is not even showing the `cursor:pointer` when interacting with it. 

@allison-c can you confirm that this is the intended? I think we want to change it right? 

if we go with the RawButton you simple need to do in your `*.style.js` file the following:

```js
import RawButton from "@paprika/raw-button";
export const ArrowItem = styled(RawButton)`
  ...yourStuff
`
```

the previous should add all the accessibility to the number that should behave like a button

Reply:
I agree we should use `RawButton` and `a11yText` prop, the cursor will be the correct one if you do it